ci-operator/config/openstack-k8s-operators/swift-operator/openstack-k8s-operators-swift-operator-main.yaml

```diff
@@ -66,6 +66,18 @@ tests:
     env:
       CEPH_HOSTNETWORK: "false"
     workflow: openstack-k8s-operators-test-build-deploy
+- as: swift-operator-build-deploy-kuttl
+  cluster_claim:
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.12"
+  steps:
+    env:
+      CEPH_HOSTNETWORK: "false"
+    workflow: openstack-k8s-operators-test-build-deploy-kuttl
```

Comment:
You are missing the container build part. e.g.: https://github.com/openshift/release/blob/e84ac90cea825395cc3f0c464f53b706b37b9472/ci-operator/config/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-main.yaml#L38C9-L49

Reply:
added, thank you!